### DISCUSSION TITLE
WebTransport API available in latest chrome 97!

### DISCUSSION THREAD

**Discussion by LoudSilencex666** [geckosio/geckos.io#156] [General]
Hi @yandeu! First of all awesome work! 

I would like to know, what do you think about http3 and QUIC support. 

Chrome is ready to use new API, and as far as I know node has some small problems with openssl but custom solution can be forked already. In my opinion web rtc is great but is also hard to maintain and develop (especially when you're making game alone like me :D).

Do you plan to implement http3 into geckos? 

Greetings!

**Comment by yandeu** [comment]
Thanks 😊

Yes, I plan to implement WebTransport with Node.js 18 and use WebRTC as a fallback 👍🏻

**Reply by LoudSilencex666** [reply]
Very nice! Can't wait to test it!

**Reply by ghost** [reply]
Any update on this ?

**Reply by yandeu** [reply]
It doesn't look like it has been added to Node v18 yet.

**Reply by ghost** [reply]
https://nodejs.org/api/dgram.html  Isn't it related ?

**Comment by emansom** [comment]
Related to [#38478](https://github.com/nodejs/node/issues/38478)

**Reply by reececomo** [reply]
> Related to [#38478](https://github.com/nodejs/node/issues/38478)

Looks like there have been some modest developments in the broader community - Relevant links:
- https://github.com/fails-components/webtransport
- https://github.com/socketio/engine.io-client/issues/703
- https://github.com/uNetworking/uWebSockets/releases/tag/v20.16.0

Haven't deep-dived so not sure if this is any progress, but wanted to keep the thread alive.
